ci-operator/step-registry/p2p/create-vsphere-test-vms/p2p-create-vsphere-test-vms-commands.sh

```
@@ -0,0 +1,189 @@
+  vcenterUser="$(< "${VCENTER_AUTH_PATH}")"
+  vcenterPassword="${vcenterUser#*:}"
+  vcenterUser="${vcenterUser%%:*}"
+  printf '%s' "${vcenterUser}" > "${SHARED_DIR}/vsphere-vcenter-user-tmp"
+  printf '%s' "${vcenterPassword}" > "${SHARED_DIR}/vsphere-vcenter-password-tmp"
+  true
+)
+
+# govc picks up credentials via environment — set without xtrace
+set +x
+export GOVC_URL="https://${vcenterHost}/sdk"
+export GOVC_USERNAME="$(< "${SHARED_DIR}/vsphere-vcenter-user-tmp")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)